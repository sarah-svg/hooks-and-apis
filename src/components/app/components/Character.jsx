/* eslint-disable max-len */
import React from 'react';
import PropTypes from 'prop-types';
import style from './style.css';

const  Character = ({ id, name, quoteText }) => {
  return (
    <div className= {style.display}>
      <p>{id}</p> 
      <p>{name}</p> 

      <p>{quoteText}</p> 

    </div>
  );
};


Character.propTypes = {
  id: PropTypes.string,
  name: PropTypes.string,
  quoteText: PropTypes.string,
};

export default Character;


