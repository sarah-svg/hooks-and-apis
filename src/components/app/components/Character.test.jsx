import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Character from './Character';

describe('character conponent', () => {
  afterEach(() => cleanup());

  it('renders the character conatinor', () => {
    const { asFragment } = render(<Character />);
    expect(asFragment()).toMatchSnapshot();

  });
});
