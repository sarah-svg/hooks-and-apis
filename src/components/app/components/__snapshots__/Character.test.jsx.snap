// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`character conponent renders the character conatinor 1`] = `
<DocumentFragment>
  <div
    class="display"
  >
    <p />
    <p />
    <p />
  </div>
</DocumentFragment>
`;
